frame/contracts/src/wasm/prepare.rs

```diff
@@ -284,22 +297,29 @@ impl<'a, T: Config> ContractModule<'a, T> {
 	/// - checks any imported function against defined host functions set, incl. their signatures.
```

Comment:
```suggestion
```

Reply:
upper part of the doc comment could be updated as well (not available from here)